presto-main/src/main/java/com/facebook/presto/execution/CreateViewTask.java

```diff
@@ -109,21 +108,20 @@ private Analysis analyzeStatement(Statement statement, Session session, Metadata
         return analyzer.analyze(statement);
     }
 
-    private String getFormattedSql(CreateView statement)
+    public static String getFormattedSql(Statement statement, SqlParser sqlParser)
```

Comment:
This method doesn't seem to belong here (especially if you're making it public so that it can be used elsewhere). Maybe move it to SqlFormatter.